src/manipulation.js

```diff
@@ -420,7 +424,7 @@ jQuery.fn.extend({
 	},
 
 	detach: function( selector ) {
-		return this.remove( selector, true );
+		return remove( this, selector, true );
```

Comment:
There's small savings (4 btyes min+gzip) to defining `detach` immediately before `remove` and moving both of them to the top of the `jQuery.fn.extend` block, but that's about it. I'm in favor of this.